### ISSUE TITLE
java.lang.IllegalStateException: Failed to create model for minecraft:skull

### ISSUE THREAD

**Issue by DissonantSynth** [Faboslav/variants-and-ventures#20]
### Minecraft version information

1.20.1

### Mod loader information

Forge

### Mod loader version information

Forge 47.3.0

### Mod version information

1.0.3

### Reproduction Steps

Attempt to run Minecraft from launcher (via Curseforge). Minecraft opens and Forge starts working. Forge states that there was an error while loading a mod. Minecraft crashes. The Minecraft launcher presents the following error:
```
The game crashed whilst rendering overlay
Error: java.lang.IllegalStateException: Failed to create model for minecraft:skull
Exit Code: -1
```

### Crash Report and latest.log

[crash-2024-06-24_23.29.11-client.txt](https://github.com/user-attachments/files/15961455/crash-2024-06-24_23.29.11-client.txt)
[latest.log](https://github.com/user-attachments/files/15961459/latest.log)

**Comment by Faboslav** [comment]
I tried:
![Screenshot 2024-06-26 at 18 54 58](https://github.com/Faboslav/variants-and-ventures/assets/14179066/4bf5da25-11a5-491a-8d1c-8b9c458c2fc4)

Cant get the error :/, is there anything else, what is different?

**Comment by LLolos9** [comment]
Minecraft version information

1.20.1
Mod loader information

Forge
Mod loader version information

Forge 47.3.1
Mod version information

1.0.3
Reproduction Steps

Attempt to run Minecraft from launcher (via Curseforge). Minecraft opens and Forge starts working. Forge states that there was an error while loading a mod. Minecraft crashes. The Minecraft launcher presents the following error:

The game crashed whilst rendering overlay
Error: java.lang.IllegalStateException: Failed to create model for minecraft:skull
Exit Code: -1

Crash Report and latest.log

https://pastebin.com/dLkky3vV

**Comment by Faboslav** [comment]
Yeah, i checked the code multiple times and based on the code, i dont know what to fix, sadly i cant reproduce this issue :/, dont know what to do now

**Comment by Faboslav** [comment]
@LLolos9 Can you please post the latest and crash logs only with Variants & Ventures enabled? Nothing more

**Comment by DissonantSynth** [comment]
@Faboslav Hi! What more can I provide you with that would be helpful?

**Comment by LLolos9** [comment]
> @LLolos9 Can you please post the latest and crash logs only with Variants & Ventures enabled? Nothing more

Oddly... now it works by itself just fine. I was more worried when it didn't work alone so thought I show a quick crash but I guess if nothing more your mod is fine then after all lol Sorry about that then really not sure what happened.
Still problems with modded side but again it's a lot of mods so probably something too far gone or something like that.

**Comment by ReeCodes** [comment]
Getting the same issue too
https://mclo.gs/f2CGj07

**Comment by ReeCodes** [comment]
It is possible that the issue is likely due to the newest Apotheosis version. At least, for me it is. After downgrading it went away.
https://www.curseforge.com/minecraft/mc-mods/apotheosis/files/5486368

**Comment by Faboslav** [comment]
Basically, if i know which mod is causing the issue, i can try to tweak the logic on my side to make it somehow work

**Comment by ReeCodes** [comment]
You know what, the more open up the game. I change my mind, it's like 75% chance for crash, 25% chance for no crash...

**Comment by ReeCodes** [comment]
Here, I'll leave a latest.log too: https://mclo.gs/ivB5Tkx

**Comment by Faboslav** [comment]
So it seems like its some kind of "race condition", will check on that too

**Comment by Faboslav** [comment]
Should be fixed with the new 1.0.4 release :).
